gru/scripts/alignmentCoverage.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -euo pipefail
```

Comment:
adding additional error handling to all scripts -- this was useful for debugging as errors now show up in the container logs